code/modules/research/designs/biogenerator_designs.dm

```diff
@@ -26,6 +26,14 @@
 	build_path = /obj/item/reagent_containers/food/condiment/milk
 	category = list("initial","Food")
 
+/datum/design/soymilk_carton
```

Comment:
Soy dopes exist for a reason too. And there are not that many recipes that make use of soy milk anyway.